lib/learn_web/client.rb

```diff
@@ -12,7 +12,7 @@ module LearnWeb
   class Client
     attr_reader :token, :conn, :silent_output
 
-    LEARN_URL = 'https://learn.co'
+    LEARN_URL = ENV.fetch('LEARN_GEM_URL', 'https://learn.co').freeze
```

Comment:
Trying to be a bit more specific with the var name here in the off chance that somewhere else we may want to use `LEARN_URL`. Trying to be specific about where this learn url is being used.

Reply:
@aturkewi Yeah, same comment as in the other PR, but I wonder if there's a way to pass this in as a command line arg to make it really easy for people to do this. exporting an env variable seems ok, but it's just not as dev friendly as I would like it to be.

Reply:
Specifically with this gem and this URL, I think the env var is the way to go and here is an example for why:
Opening a jupyter lab calls `learn open` in a way that the user or dev cannot modify or interact with the command. By having the gem look for an env var, we now set things up so that when running the ide server locally, we can inject the correct env var into the container on creation and all labs will now use gems that are hooked up to the desired environment.
Needing to pass it in as an arg means some larger code changes.

Also, an env var seems a bit more in line with the change I think we are trying to make here. The way I'm thinking about this is not that this is so much an argument that the gem would take, but just that the gem would hit QA endpoints in a QA environment. That feels more like the kind of things we generally use env vars for (just thinking towards one day when we really do have better separation between envs...)

Reply:
Cool, cool. Fair point. 🚢

Reply:
I'm ok using `LEARN_CO_URL`